enhancements/microshift/microshift-supportability-tools.md

```diff
@@ -0,0 +1,384 @@
+[must-gather-2h98l] OUT cluster-scoped-resources/storage.k8s.io/storageclasses/topolvm-provisioner.yaml
+[must-gather-2h98l] OUT host_service_logs/
+[must-gather-2h98l] OUT host_service_logs/masters/
+[must-gather-2h98l] OUT host_service_logs/masters/NetworkManager_service.log
+[must-gather-2h98l] OUT host_service_logs/masters/crio_service.log
+[must-gather-2h98l] OUT host_service_logs/masters/kubelet_service.log
+[must-gather-2h98l] OUT host_service_logs/masters/machine-config-daemon-firstboot_service.log
+[must-gather-2h98l] OUT host_service_logs/masters/machine-config-daemon-host_service.log
+[must-gather-2h98l] OUT host_service_logs/masters/openvswitch_service.log
+[must-gather-2h98l] OUT host_service_logs/masters/ovs-configuration_service.log
+[must-gather-2h98l] OUT host_service_logs/masters/ovs-vswitchd_service.log
+[must-gather-2h98l] OUT host_service_logs/masters/ovsdb-server_service.log
+[must-gather-2h98l] OUT ingress_controllers/
+[must-gather-2h98l] OUT insights-data/
+[must-gather-2h98l] OUT monitoring/
+[must-gather-2h98l] OUT network_logs/
+[must-gather-2h98l] OUT network_logs/cluster_scale
+[must-gather-2h98l] OUT network_logs/ippools
+[must-gather-2h98l] OUT network_logs/multi_networkpolicy
+[must-gather-2h98l] OUT network_logs/net_attach_def
+[must-gather-2h98l] OUT network_logs/overlappingrangeipreservations
+[must-gather-2h98l] OUT network_logs/multus_logs/
+[must-gather-2h98l] OUT pod_network_connectivity_check/
+[must-gather-2h98l] OUT pod_network_connectivity_check/podnetworkconnectivitychecks.yaml
+[must-gather-2h98l] OUT static-pods/
+[must-gather-2h98l] OUT static-pods/kube-apiserver/
+[must-gather-2h98l] OUT web_hooks/
+[must-gather-2h98l] OUT web_hooks/mutatingwebhookconfigurations.yaml
+[must-gather-2h98l] OUT web_hooks/validatingwebhookconfigurations.yaml
+[must-gather-2h98l] OUT 
+[must-gather-2h98l] OUT sent 643 bytes  received 39,570 bytes  80,426.00 bytes/sec
+[must-gather-2h98l] OUT total size is 225,917  speedup is 5.62
+[must-gather      ] OUT namespace/openshift-must-gather-sdbgv deleted
+[must-gather      ] OUT clusterrolebinding.rbac.authorization.k8s.io/must-gather-d77nw deleted
+
+
+Reprinting Cluster State:
+When opening a support case, bugzilla, or issue please include the following summary data along with any other requested information:
+error getting cluster version: the server could not find the requested resource (get clusterversions.config.openshift.io version)
+ClusterID: 
+ClusterVersion: Installing "" for <unknown>: <unknown>
+error getting cluster operators: the server could not find the requested resource (get clusteroperators.config.openshift.io)
+ClusterOperators:
+	clusteroperators are missing
+```
+
+On top of the errors, the report is empty for many components because it
+expects certain names and namespaces to exist, which don't in MicroShift.
+
+In order to use must-gather, we would need to add MicroShift awareness to the
+tool.
```

Comment:
We may want to explain a bit more on why we're not adding MicroShift-specific functionality to must-gather.